_format/1.2/index.md

```diff
@@ -13,6 +13,8 @@ JSON:API is designed to minimize both the number of requests and the amount of
 data transmitted between clients and servers. This efficiency is achieved
 without compromising readability, flexibility, or discoverability.
 
+JSON:API can and should be combined with additional standards addressing other needs of HTTP APIs such as [OAuth 2.0](https://datatracker.ietf.org/doc/html/rfc6749) for authorization and [Open API](https://spec.openapis.org/oas/latest.html) for documentation. A server supporting JSON:API can support other API standards such as [OData](https://www.odata.org/) or [GraphQL](https://graphql.org/) on the same or other endpoints.
```

Comment:
One suggestion by putting the needs first and the available standards second. This makes it more clear what needs could be, also when a need doesn't have a clear standard (e.g. file uploads).

```suggestion
For additional needs of HTTP APIs, like authentication, documentation, file uploads, and more, JSON:API can and should be combined with additional standards such as [OAuth 2.0](https://datatracker.ietf.org/doc/html/rfc6749) for authorization and [Open API](https://spec.openapis.org/oas/latest.html) for documentation. A server supporting JSON:API can also support other API standards on the same or other endpoints, such as [OData](https://www.odata.org/) or [GraphQL](https://graphql.org/).
```

Reply:
Thanks a lot. That's a good improvement.